### DISCUSSION TITLE
VSCode Plugin for Hamilton

### DISCUSSION THREAD

**Discussion by skrawcz** [apache/hamilton#145] [Ideas]
@zilto has started a VSCode extension - https://marketplace.visualstudio.com/items?itemName=ThierryJean.hamilton

This discussion is to:

1. Get feedback.
2. Log feature requests.
3. Have discussion in threads related to feedback/requests.

**Comment by skrawcz** [comment]
CC @gravesee, since I recall you had some thoughts about making development simpler.

**Comment by skrawcz** [comment]
@zilto feel free to create a short [loom](https://www.loom.com)(they have a free tier) of what you have so far.

**Reply by skrawcz** [reply]
https://www.loom.com/share/2702f433438346de83d719b67a41e0ef

**Reply by elijahbenizzy** [reply]
Love the module-level visualization + collapsing! Also rotate is 🔥 Might be worth getting rid of the multi-edges?
